meshdb/meshdb.go

```diff
@@ -509,12 +523,29 @@ func parseContractAddressesAndTokenIdsFromAssetData(assetData []byte) ([]singleA
 			return nil, err
 		}
 		for _, assetData := range decodedAssetData.NestedAssetData {
-			as, err := parseContractAddressesAndTokenIdsFromAssetData(assetData)
+			as, err := parseContractAddressesAndTokenIdsFromAssetData(assetData, contractAddresses)
 			if err != nil {
 				return nil, err
 			}
 			singleAssetDatas = append(singleAssetDatas, as...)
 		}
+	case "ERC20Bridge":
+		var decodedAssetData zeroex.ERC20BridgeAssetData
+		err := assetDataDecoder.Decode(assetData, &decodedAssetData)
+		if err != nil {
+			return nil, err
+		}
+		tokenAddress := decodedAssetData.TokenAddress
+		// HACK(fabio): Despite Chai ERC20Bridge orders encoding the Dai address as
+		// the tokenAddress, we actually want to react to the Chai token's contract
+		// events, so we actually return it instead.
```

Comment:
@abandeali1 can you confirm that this is the intended behavior?

Reply:
Yep, the maker is actually holding Chai, not Dai, so we'll only need to watch the Chai contract's events.